smdebug/core/utils.py

```diff
@@ -84,11 +84,10 @@ def list_files_in_directory(directory, file_regex=None):
 
 
 def list_collection_files_in_directory(directory):
-    collections_directory = get_path_to_collections(directory)
     import re
 
-    collections_file_regex = re.compile(".*_?collections.json")
-    return list_files_in_directory(collections_directory, file_regex=collections_file_regex)
+    collections_file_regex = re.compile(".*_?collections.json$")
+    return list_files_in_directory(directory, file_regex=collections_file_regex)
```

Comment:
why did you remove the collections directory?

Reply:
the function call is invoked like this, so there is no need get collections directory again
`return list_collection_files_in_directory(get_path_to_collections(self.path))`

Reply:
Slightly confused as to how it was working earlier